content/blog/posts/mps-blog.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Explore Multiplayer Gaming with the O3DE Multiplayer Sample - Now Available for Download!"
+date: 2055-04-19
+slug: mps-sample 
+Author: Darrin McPherson
+blog_img: "/images/blog/mps-sample/image01.png"
+full_img: "/images/blog/mps-sample/image01.png"
+---
+
+We are happy to announce the release of the O3DE Multiplayer Sample Game (MPS). The MPS is a fully open source mini game where players battle one another for dominance in a multi-tiered, under-construction, Starbase. This new sample game is intended to serve as an example for the O3DE community to use in the creation of other games as well as to demonstrate a number of features in O3DE, including networking, material canvas, asset creation best practices, and the setup of visual effects setup using the [PopcornFX gem](https://github.com/PopcornFX/O3DEPopcornFXPlugin), and audio effects using the [WWise gem](https://github.com/o3de/o3de/tree/development/Gems/AudioEngineWwise). 
+
+The design intent of the MPS was to create something that would be fun to play, feature exciting exploration that rewarded player mastery, have clear risk vs. reward elements, and feature both player vs. environment, and player vs. player gameplay. We also wanted it to be highly configurable and expandable so that the O3DE community continue to iterate and add to it. From a technical angle, we also wanted a multi-faceted game that would provide a platform to test against when adding new systems to O3DE.
```

Comment:
Would argue that theres no real depth, so "player mastery" seems to be a stretch.

Reply:
I'd argue that learning the map, where gem placements are, which gems appear in each round and how to navigate through the level using teleporters and jump pads is player mastery. The more familiar you are with these things makes you a better player, outside of just having innate FPS shooter skills.